boards/st/stm32c0116_dk/stm32c0116_dk.dts

```diff
@@ -41,6 +42,38 @@
 			label = "reset button";
 			gpios = <&gpiof 2 GPIO_ACTIVE_LOW>;
 			status = "okay";
+			zephyr,code = <INPUT_KEY_0>;
+		};
+	};
+
+	adc-keys {
+		compatible = "adc-keys";
+		io-channels = <&adc1 8>;
+		keyup-threshold-mv = <3300>;
+
+		select_key {
+			press-thresholds-mv = <0>;
+			zephyr,code = <INPUT_KEY_1>;
```

Comment:
You can use more relevant key codes, see e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/boards%2Fseeed%2Fwio_terminal%2Fwio_terminal.dts#L75-L99

Reply:
Thanks for reference: it's done